bb-new.html

```diff
@@ -0,0 +1,296 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>BlackBerry Page</title>
+    <!-- Bootstrap CSS -->
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-BmbxuPwQa2lc/FVzBcNJ7UAyJxM6wuqIj61tLrc4wSX0szH/Ev+nYRRuWlolflfl" crossorigin="anonymous">
+    <link href="styles.css" rel="stylesheet">
+  </head>
+  <body>
+    <div class="container-fluid">
+      <div class="row">
+        <div class="col">
+          <header>
+            <nav>                       
+              <ul>
+                <li class="nav-list-item"><a href="#home">Logo</a></li>
+                <li class="nav-list-item"><a href="#products">Products</a></li>
+                  <div class="dropdown">
+                    <a class="dropdown-item" href="#">Unified Endpoint Security/a>
```

Comment:
Tag is not properly closed
You can see in the browser, when you open this page up, how this missing tag (and possibly other issues) breaks your entire navigation.  When this happens, you have to go back to the navigation and find where things break.